Scripts/GUI.js

```diff
@@ -76,14 +76,44 @@ export default function GUI(inModel){
       incidentPhiEnvelope = addEnvelopeControl(ptLightSliderDiv, "φ",
         "slider_incidentPhi", -180, 180, starting_phi);
 
-      let camRotSlider = document.getElementById("slider_camRot");
-      camRotSlider.setAttribute("min", -180);
-      camRotSlider.setAttribute("max", 180);
-      camRotSlider.setAttribute("step", 1);
-      camRotSlider.setAttribute("value", 0);
+      // let camRotSlider = document.getElementById("slider_camRot");
+      // camRotSlider.setAttribute("min", -180);
+      // camRotSlider.setAttribute("max", 180);
+      // camRotSlider.setAttribute("step", 1);
+      // camRotSlider.setAttribute("value", 0);
+    },
+
+    setupButtonCallback = function(button, url) {
+      // Buttons to select predefined BRDFs
+      button.on('click', (event) => {
+        var blob = null;
+        var xhr = new XMLHttpRequest();
+        xhr.open("GET", url);
+        xhr.responseType = "blob";//force the HTTP response, response-type header to be blob
+        xhr.onload = function()
+        {
+          if (this.status == 200) {
+            // Note: .response instead of .responseText
+            var blob = new Blob([this.response], {type: 'Blob'});
+
+            model.loadAnalyticalBRDF([blob]).then(returnResult => {
+              //console.log(returnResult);
+              const {uniforms, uniform_update_funcs} = returnResult;
+              spawnUniformSliders(uniforms, uniform_update_funcs, brdfSliderDiv,
+                brdfCheckboxDiv);
+            });
+          }
+        }
```

Comment:
[JSHint](http://jshint.com/) complains about a missing semicolon here. I.e the code should look like:
```javascript
xhr.onload = function()                                                                        
{
...
};
```
Some people may not be using semicolons in their JS but most of the rest of the code does use them religiously, so I'm guessing we should probably do it here as well.

Reply:
@n8vm @DQLin Should we all be using a common syntax checking tool for the sake of consistency?